server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -7,7 +7,7 @@ import { ISession, NetworkError } from "@fluidframework/server-services-client";
 import { IDocument, ICollection } from "@fluidframework/server-services-core";
 import { getLumberBaseProperties, Lumberjack } from "@fluidframework/server-services-telemetry";
 
-const defaultSessionStickinessDurationMs = 60 * 60 * 1000; // 60 minutes
+const defaultSessionStickinessDurationMs = 180 * 60 * 1000; // 180 minutes
```

Comment:
I think we can leave the default at 1 hour for the same reasons as @pradeepvairamani's message. We can/will configure this within AFR as needed.

Reply:
Thanks. Done.